lib/MetaCPAN/Script/Suggest.pm

```diff
@@ -0,0 +1,117 @@
+package MetaCPAN::Script::Suggest;
+
+use strict;
+use warnings;
+
+use Log::Contextual qw( :log );
+use Moose;
+use MetaCPAN::Types qw( Bool Int );
+use DateTime;
```

Comment:
Could we alpha-sort these?

Also, it's nice to be explicit about not importing anything.  ie `use DateTime ()`.